internal/cis/client.go

```diff
@@ -52,7 +51,6 @@ func NewClient(ctx context.Context, config Config, log logrus.FieldLogger) *Clie
 	return &Client{
 		httpClient: httpClientOAuth,
 		config:     config,
-		log:        log.WithField("client", "CIS-2.0"),
```

Comment:
Client version won't be visible in logs. This may cause trouble while debugging issues.